examples/gke-security-best-practices/gke.tf

```diff
@@ -0,0 +1,136 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# set the latest gke version
+data "google_container_engine_versions" "cluster" {
+  zone = "${var.zone}"
+}
+
+# create our GKE cluster
+resource "google_container_cluster" "cluster" {
```

Comment:
Can you use the GKE module instead? https://github.com/terraform-google-modules/terraform-google-kubernetes-engine

We want to consolidate best practices and standard GKE module configs.

Reply:
agreed, will update to reflect